rules/external_project_access_rules.yaml

```diff
@@ -14,5 +14,6 @@
 
 rules:
 - name: Only allow access to projects in my organization.
-  allowed_ancestors: 
-  - organizations/{ORGANIZATION_ID}
+  allowed_ancestors:
+  - organizations/660570133860
+  #{ORGANIZATION_ID}
```

Comment:
temporarily assigned an organization ID to pass the regex statement. will need to replace back with {ORGANIZATION_ID} when ready to merge